internal/kibana/packages.go

```diff
@@ -76,12 +76,20 @@ func processResults(action string, statusCode int, respBody []byte) ([]packages.
 	}
 
 	var resp struct {
-		Assets []packages.Asset `json:"response"`
+		// Assets are here when old API is used.
+		Response []packages.Asset `json:"response"`
+
+		// Assets are here when new API is used.
+		Items []packages.Asset `json:"items"`
```

Comment:
Just a small suggestion. But maybe it's not needed because it's already in the `epmPackageURL`. 

```suggestion
		// Assets are here when old API (kibana < 8.0) is used.
		Response []packages.Asset `json:"response"`

		// Assets are here when new API (kibana >= 8.0) is used.
		Items []packages.Asset `json:"items"`
```

Reply:
:+1: